components/CommunityInvolvement.vue

```diff
@@ -1,12 +1,13 @@
 <template>
-  <div class="section flex flex-wrap justify-center">
+  <section class="section flex flex-wrap justify-center">
     <div class="section-content">
-      <h3>We ❤️ the Ember.js Community</h3>
+      <h2>We ❤️ the Ember.js Community</h2>
```

Comment:
With this being changed to an `h2`, I think we might need to add some classes to change the text size?